test/treat-locks/common.lock.test.js

```diff
@@ -16,16 +16,23 @@
 // under the License.argv._
 
 const commonLock = require('../../lib/treat-locks/common.lock');
+const NpmOptions = require('../../lib/treat-locks/npm.options');
 
-test('Verify yarnlock host replaced with final slash', () => {
+test('Verify register has an appended final slash when input does not have a final slash', () => {
```

Comment:
this test should be moved to a new npm.options.test.js since it does belong to npm.options.js functionality

Reply:
Alright done